src/backend/rest.rs

```diff
@@ -56,6 +56,21 @@ impl Display for RestErrorCode {
     }
 }
 
+#[derive(PartialEq, Eq)]
+enum RestMethod {
```

Comment:
Fyi I added this enum to avoid doing something like:
```rust
Ep::METHOD as *const fn(&str) -> ureq::Request == ureq::get as *const fn(&str) -> ureq::Request
```

Reply:
Seems reasonable. But they should be called `Get` and `Post` as Clippy demands

Reply:
Sure, renamed, thanks!